build-tools/pnpm-lock.yaml

```diff
@@ -11897,6 +12440,13 @@ packages:
     requiresBuild: true
     dev: true
 
+  /yarn/1.22.22:
+    resolution: {integrity: sha512-prL3kGtyG7o9Z9Sv8IPfBNrWTDmXB4Qbes8A9rEzt6wkJV8mUvoirjU0Mp3GGAU06Y0XQyA3/2/RQFVuK7MTfg==}
+    engines: {node: '>=4.0.0'}
+    hasBin: true
+    requiresBuild: true
+    dev: false
+
```

Comment:
Something now pulls in yarn? For what?

Reply:
The new patch of @oclif/plugins-plugins

![image](https://github.com/microsoft/FluidFramework/assets/716334/0dee614e-be00-46de-b079-6cd96a4e033d)

[From a previous patch](https://github.com/oclif/plugin-plugins/releases/tag/5.0.11). Seems reasonable based on what oclif is...?

Reply:
It's part of the default oclif setup, but we don't currently use it. I'll remove the dep and the config in package.json. We can always add it back if we want to use the feature later.